wire-grpc-client/src/jvmMain/kotlin/com/squareup/wire/internal/grpc.kt

```diff
@@ -118,7 +118,7 @@ internal fun <R : Any> SendChannel<R>.readFromResponseBodyCallback(
               }
               exception = response.grpcResponseToException()
             } catch (e: IOException) {
-              exception = response.grpcResponseToException(e)
+              exception = response.grpcResponseToException(false, e)
```

Comment:
intentionally only did this for unary calls because there's a specific test to disallow this for streaming (`duplexSuspend_responseBodyThrows`) - i suspect this is because when something goes wrong, the channel is explicitly closed, therefore making it not possible to read the trailers.